src/client/routes/login/PasswordLogin.tsx

```diff
@@ -29,21 +26,23 @@ interface Props {}
  */
 export const validateAndSubmitLogin = (
 	username: string,
-	password: string,
-	setInvalidFn: React.Dispatch<React.SetStateAction<boolean>>
+	password: string
+	// setInvalidFn: React.Dispatch<React.SetStateAction<boolean>>
 ): void => {
 	const emailValid = emailValidation(username);
 	const passValid = passwordValidation(password);
 	// Do one more check for valid fields (to handle edge case where
 	// constructor sets valids to true)
+	if (emailValid && passValid) {
+		// dummy
```

Comment:
What does this mean?

Reply:
Just to make eslint stop complaining about unused vars

Reply:
This method is easy to find and work on later

Reply:
Why do we need the vars at all? Can't we just check if they're valid and not store anything?

Reply:
We might to use them in multiple if-statements or whatever, that if-check is just a stub

Reply:
ah ok - can you someone add a fixme comment then this should be good